benchmarks/web-serving/db_server/Dockerfile

```diff
@@ -1,30 +1,12 @@
-FROM ubuntu:14.04
+FROM cloudsuite/mysql:mariadb-10.3
 LABEL maintainer="Mark Sutherland <mark.sutherland@epfl.ch>"
 
 ENV root_password root
-ENV web_host web_server 
 
-# Install MySQL
-RUN echo mysql-server mysql-server/root_password password ${root_password} | sudo debconf-set-selections
-RUN echo mysql-server mysql-server/root_password_again password ${root_password} | sudo debconf-set-selections
```

Comment:
I think the above two lines should be changed following the changes with mariadb as well.

Reply:
I am sorry , i didn't understand what to change here, these lines are already removed.

Reply:
Sorry, never mind.